hr_timesheet_task_stage/oca_dependencies.txt

```diff
@@ -0,0 +1 @@
+project https://github.com/OCA/project project_stage_closed
```

Comment:
You only need this:

```suggestion
project
```

Reply:
ok

Reply:
But... you have to upload the change.

Also, this file must be in the repo root, not in the addon folder.